documentation/src/docbkx/chapter41-configuration-file.xml

```diff
@@ -437,15 +461,15 @@
 				&lt;reference-data-catalog&gt;
 				&#160;&#160;&lt;synonym-catalogs&gt;
 				&#160;&#160;&#160;...
-				&#160;&#160;&#160;&#160;&lt;text-file-synonym-catalog name="textfile_syn" description="My text file synonyms"&gt;
+				&#160;&#160;&#160;&#160;&lt;text-file-synonym-catalog
+				name="textfile_syn" description="My text file synonyms"&gt;
```

Comment:
This is an unfortunate reformatting because it happens in <programlisting> where newlines are actually significant!

Normally I use &#160; instead of space to avoid this. Not sure why it wasn't applied here in the first place.